src/Core/Core.csproj

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="12.0" DefaultTargets="Build;RunGrunt" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
@williambogaiv Ordering in the **Core.csproj** addressed in 86c88d2c390001bd8ed6cb3096c62688f94459e1. **build.cmd** appears to run grunt.